spring-cloud-starter-stream-sink-field-value-counter/README.adoc

```diff
@@ -1,12 +1,22 @@
 //tag::ref-doc[]
 = Field Value Counter Sink 
 
-A field value counter is a Metric used for counting occurrences of unique values for a named field in a message payload. This sinks supports the following payload types out of the box:
+A field value counter is a Metric used for counting occurrences of unique values for a named field in a message payload.
 
-* POJO (Java bean)
-* Map
-* Tuple
-* JSON String
+== Input
+
+=== Headers
+
+`content-type: application/octet-stream`
+
+=== Payload
+
+This sinks supports the following payload types out of the box:
+
+* `POJO (Java bean)`
+* `Map`
+* `Tuple`
+* `JSON String`
```

Comment:
I think we should describe `content-type` header for all these valid payload types.
when it is `String`, the content-type should be `text/plain` or `application/json`.
And seems for me there is a special content type for the `Tuple`.